api/src/main/java/net/wesjd/anvilgui/AnvilGUI.java

```diff
@@ -27,6 +27,7 @@
 import org.bukkit.inventory.ItemStack;
 import org.bukkit.inventory.meta.ItemMeta;
 import org.bukkit.plugin.Plugin;
+import org.geysermc.geyser.api.GeyserApi;
```

Comment:
won't this import break all usages of the library on servers without the geyser plugin installed?

i believe you will need a separate class like the wrapper classes and you only instantiate it if geyser is installed

Reply:
That was not the case in my testing on a 1.21 Paper server with the only plugins being Geyser, Floodgate, and a modified version of [Quests](https://github.com/PikaMug/Quests) with this PR shaded in. Removing Geyser and Floodgate did not affect the ability to open AnvilGUIs.

Reply:
sweet